todo_app/app.py

```diff
@@ -1,11 +1,36 @@
-from flask import Flask
+from flask import Flask, render_template, session, request, redirect, url_for
 
 from todo_app.flask_config import Config
 
+from todo_app.data.session_items import get_items
+from todo_app.data.session_items import add_item
```

Comment:
Down to preference, but note we could combine these like you have the flask imports:
```suggestion
from todo_app.data.session_items import get_items, add_item
```